spec/controllers/v1/legislators_controller_spec.rb

```diff
@@ -76,4 +76,15 @@
       end
     end
   end
+
+  describe "GET targeted" do
+    it "returns a hash with a targeted legislator" do
+      legislator = FactoryGirl.create(:representative, :targeted)
+
+      get :targeted
+      json_response = JSON.parse(response.body)
+
+      expect(json_response[0]['name']).to eq(legislator.name)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.